src/components/SliderContent.tsx

```diff
@@ -1,40 +1,97 @@
-import React, { FC } from 'react'
+import React, { useEffect } from 'react'
 import { Dimensions, ImageSourcePropType, Image } from 'react-native'
 import { Box, Text } from 'utils/theme/index'
-import { isSmallScreen } from 'utils/deviceSizes'
+import { isSmallScreen, windowHeight } from 'utils/deviceSizes'
+import Animated, {
+  useSharedValue,
+  withTiming,
+  useAnimatedStyle,
+  withDelay,
+  withSpring,
+  withRepeat,
+} from 'react-native-reanimated'
 
 const { width } = Dimensions.get('window')
 
 type SliderContentProps = {
   title: string
   text: string
   image: ImageSourcePropType
+  isUserLoggedIn: boolean
+  disableInitialAnimation?: boolean
 }
 
+const AnimatedBox = Animated.createAnimatedComponent(Box)
+const IMAGE_HEIGHT = 300
+const middleScreenY = windowHeight / 2 - IMAGE_HEIGHT / 1.7
+
 const imgStyles = {
-  width: '75%',
-  maxWidth: 240,
+  height: IMAGE_HEIGHT,
 }
 
-export const SliderContent: FC<SliderContentProps> = ({ title, text, image }) => (
-  <Box width={width} alignItems="center" justifyContent="space-around">
-    <Box
-      justifyContent="center"
-      alignItems="center"
-      aspectRatio={isSmallScreen ? 1.4 : 1}
-      width="100%"
-      backgroundColor="primary"
-      marginTop="-ml"
-      marginBottom="-ml">
-      <Image style={imgStyles} source={image} resizeMode="contain" />
-    </Box>
-    <Box maxWidth="80%" justifyContent="center" alignItems="center">
-      <Text variant="title1" paddingBottom="m" color="alwaysBlack">
-        {title}
-      </Text>
-      <Text variant="body1" color="alwaysBlack">
-        {text}
-      </Text>
+export const SliderContent = ({
+  title,
+  text,
+  image,
+  isUserLoggedIn,
+  disableInitialAnimation,
+}: SliderContentProps) => {
+  const translateY = useSharedValue(middleScreenY)
+  const scale = useSharedValue(0.57)
+  const opacity = useSharedValue(0)
+  const rotate = useSharedValue(0)
```

Comment:
I'm wondering if you can create only 1 shared value and then use interpolation 🤔